src_c/include/pgcompat.h

```diff
@@ -102,7 +101,6 @@ typedef uint8_t Uint8;
  * Ref: https://wiki.libsdl.org/SDL_AudioDeviceEvent
  */
 #define SDL2_AUDIODEVICE_SUPPORTED
```

Comment:
Since this will be always defined now, this macro can be removed, and all guards for this should be allowed

Reply:
Yeah good catch once again.